build/tutorials/tutorials-contest/avalanche-erc721-tutorial/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Remove the license. Everything in the docs repos is under the [BSD 3-Clause license](https://github.com/ava-labs/avalanche-docs/blob/master/LICENSE)